recipes/valgrind/files/arm-cortex-support.patch

```diff
@@ -0,0 +1,29 @@
+Valgrind's configure script checks for ${host_cpu} starting with
+armv7, but we call it just arm. Since Cortex A chips are (at least)
+ARMv7, make the configure script recognize arm-cortexa*.
+
+Origin: OE-lite (first introduced in 5b66593c)
+Upstream-status: Inappropriate
```

Comment:
is this just what you think, or have we checked with upstream? not all upstreams are refusing changes like this :)

if we just don't want to spend time on trying to upstream it, which of course can be reasonable, then we should write something else there